src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import './mixins';
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  height: 100vh;
+  width: 100vw;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    bottom: 50vmin;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    z-index: 1;
+
+    @include rotate-animation(3600s, steps(60));
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    bottom: 50vmin;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    z-index: 2;
+
+    @include rotate-animation(60s, linear);
+  }
+
+  &__middle {
+    position: absolute;
+    width: 5vmin;
+    height: 5vmin;
+    border-radius: 2.5vmin;
+    background-color: #f6a603;
+    z-index: 3;
+  }
+
+  &--speed-up {
+    .stopwatch__minutes-hand {
+      @include rotate-animation(600s, steps(60));
+    }
+
+    .stopwatch__seconds-hand {
+      @include rotate-animation(10s, linear);
+    }
```

Comment:
The `--speed-up` modifier should correctly override the animation duration for the `.stopwatch__minutes-hand` and `.stopwatch__seconds-hand`. Confirm that the mixin allows for this override and that the new values are appropriate for the 'speed-up' feature.